cypress.json

```
@@ -1,5 +1,6 @@
 {
   "video": false,
+   "chromeWebSecurity": false,
```

Indent slightly off.
```suggestion
  "chromeWebSecurity": false,
```
Also, I'd like to understand why we need to do this, as it seems like it could make us insensitive to some bugs we might want Cypress to help us know about. Is it because of the clicking on "Find out what's changed"?

---

Have just tried running locally without it, this is what happens:

```
     CypressError: Cypress detected a cross origin error happened on page load:

  > Blocked a frame with origin "http://localhost:7080" from accessing a cross-origin frame.

Before the page load, you were bound to the origin policy:
  > http://localhost:7080

A cross origin error happens when your application navigates to a new superdomain which does not match the origin policy above.

This typically happens in one of three ways:

1. You clicked an <a> that routed you outside of your application
2. You submitted a form and your server redirected you outside of your application
3. You used a javascript redirect to a page outside of your application

Cypress does not allow you to change superdomains within a single test.

You may need to restructure some of your test code to avoid this problem.

Alternatively you can also disable Chrome Web Security which will turn off this restriction by setting { chromeWebSecurity: false } in your 'cypress.json' file.

https://on.cypress.io/cross-origin-violation

      at Object.cypressErr (http://localhost:7080/__cypress/runner/cypress_runner.js:65283:11)
      at Object.throwErr (http://localhost:7080/__cypress/runner/cypress_runner.js:65248:18)
      at Object.throwErrByPath (http://localhost:7080/__cypress/runner/cypress_runner.js:65275:17)
      at onPageLoadErr (http://localhost:7080/__cypress/runner/cypress_runner.js:55386:21)
      at HTMLIFrameElement.<anonymous> (http://localhost:7080/__cypress/runner/cypress_runner.js:61113:19)
      at HTMLIFrameElement.dispatch (http://localhost:7080/__cypress/runner/cypress_runner.js:17523:27)
      at HTMLIFrameElement.elemData.handle (http://localhost:7080/__cypress/runner/cypress_runner.js:17335:28)
```

Doesn't look like there's a good alternative tbh.

---

Do we know which navigation causes the error? If it's navigating by clicking on an `<a>` do we actually need to do that? Advice here https://docs.cypress.io/guides/guides/web-security.html#External-Navigation suggests asserting the existence of/target of the link rather than clicking.

---

Hmmm but we do two things with that - when we click on the link, we set a cookie, which we can then check the value of. If we only assert the `href` of the link, we can't assert that clicking it sets the cookie we expect.

---

I see what you mean. It just seems a bit of a shame to permanently disable web security in the test suite just to test setting a cookie, which is not even the actual cause of the error. Is there an easy way we could stub out/remove the href instead? Or have the event handler return false? Just spitballing...